api/graphql/mutations.js

```diff
@@ -83,18 +83,16 @@ export function updateMembership (userId, { id, data }) {
 
   return Membership.query().where({id, user_id: userId})
   .update(whitelist)
-  .returning('id')
-  .then(ids => Membership.where('id', ids[0]).fetch())
+  .then(() => ({success: true}))
```

Comment:
The front-end already ignores the response, so I figured there was no need to provide one. It's not a change for mutations in general, just ones where the front-end doesn't need any data about the outcome.